src/__tests__/Func.test.js

```diff
@@ -0,0 +1,5 @@
+import { getCrypto } from '../redux/cryptoSlice';
+
+it('getCrypto should return a promise', () => {
+  expect(getCrypto()).toEqual(expect.any(Function));
+});
```

Comment:
- [ ] Kindly add some unit tests for all of your reducers. Reducers are pure functions by default so testing them is fairly easy and does not require mocking. You can have a stubbed action with a hard coded state and use it for your tests.